keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/CombinedRepository.kt

```diff
@@ -357,10 +358,10 @@ class CombinedRepository(
   override fun applicationEventHistory(application: String, downTo: Instant): List<ApplicationEvent> =
     resourceRepository.applicationEventHistory(application, downTo)
 
-  override fun resourceEventHistory(id: String, limit: Int): List<ResourceEvent> =
+  override fun resourceEventHistory(id: String, limit: Int): List<ResourceHistoryEvent> =
```

Comment:
Again, the resource event history is now composed of events implementing `ResourceHistoryEvent`.